copulas/univariate/student_t.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from scipy.stats import t
+
+from copulas.univariate.base import BoundedType, ParametricType, ScipyModel
+
+
+class StudentTUnivariate(ScipyModel):
+    """Wrapper around scipy.stats.t.
+
+    Documentation: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.t.html
+    """
+
+    PARAMETRIC = ParametricType.PARAMETRIC
+    BOUNDED = BoundedType.UNBOUNDED
+
+    MODEL_CLASS = t
+
+    def _fit_constant(self, X):
+        self._params = {
+            'df': 100,
```

Comment:
I think that this `100` value will not be consistent across different constant value.
For example, a vector which is not constant but very close to 0, like `[0.00001, 0, 0....]` gets a `df` value of `~2` while, a vector which is very close to 1, like `[1.00001, 1, 1, ...]` gets a `df` value of `~39`.

However, `scipy.stats.t` DOES handle constant arrays as input, so let's take advantage of it doing the following:
1. Remove this method completely from this class, so there is no `_fit_constant`.
2. Make `_is_constant` always return `False`.
3. Also overwrite `_check_constant_value` to always return `False`.
4. Adapt the tests accordingly.

```python
    def _is_constant(self):
        return False

    def _check_constant_value(self, *args, **kwargs):
        return False
```

This way, the our class will never handle constant values on its own and we will be always delegating to scipy.

Reply:
committed solution is failing a bunch of test which were previously ok. Could you have a look and check if the change are in line with what you meant? Thank you!

Reply:
Nevermind I sorted it out and fixed all tests.
Removing `_fit constant` generated some issue in downstream tests. So now `_fit_constant` simply returns `_fit` and the every testing issues is circumvented whilst the handling of constant fitting is properly delegated to `scipy.stats.t` as recommended